cpg-core/src/test/resources/function-dfg.json

```diff
@@ -0,0 +1,56 @@
+[
```

Comment:
Can we also support yaml files? Or maybe instead of JSON? JSON seems a little bit bloaty here I think

Reply:
We now support json and yml